src/components/container/ExternalAccountsPreview/ExternalAccountsPreview.tsx

```diff
@@ -64,7 +65,11 @@ export default function (props: ExternalAccountsPreviewProps) {
         if (props.applicationUrl == "preview") {
             console.log("PREVIEW: Opening the external accounts application.");
         } else {
-            MyDataHelps.openApplication(props.applicationUrl);
+            if (props.onClick) {
```

Comment:
Could also remove the "preview" here and use the click handler for stories..

Reply:
Done